File: game_service/src/main/java/com/devxijn/game_service/entity/Match.java
package com.devxijn.game_service.entity;

import lombok.*;
import lombok.experimental.FieldDefaults;

import java.util.List;

/**
 * @author devxijn
 * @since 7/20/2025
 */
@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class Match {
    List<Pair> pairRows;
    List<Pair> pairColumns;
}

File: match-3-client/src/api/models/Match.ts
import { Pair } from '../../types/Pair'

export class Match {
  pairRows: Pair
  pairColumns: Pair
  constructor(pairRows: Pair, pairColumns: Pair) {
    this.pairRows = pairRows
    this.pairColumns = pairColumns
  }
}
